layers/meta-balena-5x-owa/recipes-core/pmsrv/files/pmsrv.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=pmsrv service, a service to open a socket comms with the onboard uprocessor and reset the HW watchdog.
+DefaultDependencies=no 
+
+[Service]
+ExecStart=/usr/bin/pmsrv
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+Alias=owasysd-pmsrv.service
```

Comment:
I see you have the service already in the BSP layer? https://github.com/Owasys/meta-owasys/blob/master/meta-owa5x/recipes-core/pmsrv/files/pmsrv.service

Reply:
Yes, but they are different services, mind the `DefaultDependencies=no`change

Reply:
And this change would be specifically for balenaOS? Would it not be something you'd require on every OS, to have the service started as soon as possible?

Reply:
We got a problem when removing default dependencies on our system that seemed to be no on balenaOS. I am trying another approach so hold on, I'll may include a similar change on our meta-owasys.